docs/how-to/hip_runtime_api/memory_management/unified_memory.rst

```diff
@@ -17,12 +19,13 @@ and promise increased efficiency and innovation.
 
 Unified memory
 ==============
+
 Unified Memory is a single memory address space accessible from any processor
 within a system. This setup simplifies memory management processes and enables
 applications to allocate data that can be read or written by code running on
 either CPUs or GPUs. The Unified memory model is shown in the following figure.
 
-.. figure:: ../data/unified_memory/um.svg
+.. figure:: ../../../data/how-to/hip_runtime_api/memory_management/unified_memory/um_old.svg
```

Comment:
Update the picture. Get rid of the old one, don't mention APUs specifically, but explain the whole concept

Reply:
Replaced.